code/_core/obj/item/clothing/hats/paperbag.dm

```diff
@@ -20,22 +20,80 @@
 
 	dyeable = TRUE
 
-	blocks_clothing = SLOT_FACE_WRAP | SLOT_HEAD | SLOT_FACE
+	blocks_clothing = SLOT_FACE_WRAP | SLOT_HEAD
+
+/obj/item/clothing/head/hat/paperbag/save_item_data(var/save_inventory = TRUE)
+	. = ..()
+	SAVEVAR("logo")
+	return .
+
+/obj/item/clothing/head/hat/paperbag/load_item_data_pre(var/mob/living/advanced/player/P,var/list/object_data)
+	. = ..()
+	LOADVAR("logo")
+	update_sprite()

```

Comment:
update_sprite should be called in `Finalize()` instead.

Reply:
is this good now i dont have a big brain

Reply:
okay nevermind i dont even need to call anything to update or finalize the sprite, it loads it up properly without